Tests/Enum/main.cpp

```diff
@@ -140,5 +142,24 @@ int main (int argc, char* argv[])
         AMREX_ALWAYS_ASSERT(my_green == MyColor2::green);
     }
 
+
+    {
+        auto const& kv = amrex::getEnumNameValuePairs<Location>();
+        amrex::Print() << "Name : Value\n";
+        for (auto const& item : kv) {
+            amrex::Print() << "  " << item.first << ": "
+                           << static_cast<int>(item.second) << "\n";
+        }
+        AMREX_ALWAYS_ASSERT(static_cast<int>(Location::entry) == 1);
+        AMREX_ALWAYS_ASSERT(static_cast<int>(Location::exit) == -1);
+        AMREX_ALWAYS_ASSERT(static_cast<int>(Location::after_exit) == 0);
```

Comment:
Just to cover this function, too:
```suggestion
        AMREX_ALWAYS_ASSERT(static_cast<int>(Location::entry) == 1);
        AMREX_ALWAYS_ASSERT(static_cast<int>(Location::exit) == -1);
        AMREX_ALWAYS_ASSERT(static_cast<int>(Location::after_exit) == 0);
        AMREX_ALWAYS_ASSERT(amrex::toUnderlying(Location::entry) == 1);
        AMREX_ALWAYS_ASSERT(amrex::toUnderlying(Location::exit) == -1);
        AMREX_ALWAYS_ASSERT(amrex::toUnderlying(Location::after_exit) == 0);
```